resources/validations/ui/src/use-cases/validate-ssp-xml.test.ts

```diff
@@ -0,0 +1,5 @@
+describe('validate ssp use case', () => {
+  it('works', () => {
+    expect(true).toBe(true);
```

Comment:
NB: If this fails, we all fail! 😆 

Jokes aside, I presume more tests to follow after MVP status?

Reply:
Yes, trying to fill them in 👍